test/helper.rb

```diff
@@ -50,6 +50,21 @@ def change_params(change)
 end
 
 class ActionController::TestCase
+  if RUBY_VERSION>='2.6.0'
+    if Rails.version < '5'
+      class ActionController::TestResponse < ActionDispatch::TestResponse
```

Comment:
Shouldn’t this be `class ::ActionController::TestResponse`? Or better, reopen the `ActionController` module and define `TestCase` and `TestResponse` classes independent of each other.

Reply:
We cleaned this up in d307af4. Can you take another look?